python/test/block/Test_ACEtk_TabulatedOutgoingEnergyDistribution.py

```diff
@@ -0,0 +1,61 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from ACEtk import TabulatedOutgoingEnergyDistribution
+
+class Test_ACEtk_TabulatedOutgoingEnergyDistribution( unittest.TestCase ) :
+    """Unit test for the TabulatedOutgoingEnergyDistribution class."""
+
+    chunk = [                32,                  3,  1.00000000000E-11,  1.00000000000E+00,
+              2.00000000000E+01,  0.50000000000E+00,  0.50000000000E+00,  0.50000000000E+00,
+              0.00000000000E+00,  0.50000000000E+00,  1.00000000000E+00 ]
+
+    def test_component( self ) :
+
+        def verify_chunk( self, chunk ) :
+
+            # verify content
+            self.assertEqual( False, chunk.empty )
+            self.assertEqual( 11, chunk.length )
+            self.assertEqual( "DLW::TabulatedOutgoingEnergyDistribution", chunk.name )
+
+            self.assertEqual( 2.1, chunk.incident_energy )
+            self.assertEqual( 2, chunk.interpolation )
+            self.assertEqual( 3, chunk.number_discrete_photon_lines )
+            self.assertEqual( 3, chunk.number_outgoing_energies )
+
+            self.assertEqual( 3, len( chunk.outgoing_energies ) )
+            self.assertAlmostEqual( 1e-11, chunk.outgoing_energies[0] )
+            self.assertAlmostEqual( 20., chunk.outgoing_energies[-1] )
+
+            self.assertEqual( 3, len( chunk.pdf ) )
+            self.assertAlmostEqual( .5, chunk.pdf[0] )
+            self.assertAlmostEqual( .5, chunk.pdf[-1] )
+
+            self.assertEqual( 3, len( chunk.cdf ) )
+            self.assertAlmostEqual( 0., chunk.cdf[0] )
+            self.assertAlmostEqual( 1., chunk.cdf[-1] )
+
+            # verify the xss array
+            xss = chunk.xss_array
+            for index in range( chunk.length ) :
+
+                self.assertAlmostEqual( self.chunk[index], xss[index] )
+
+        # the data is given explicitly
+        chunk = TabulatedOutgoingEnergyDistribution(
+                    incident = 2.1,
+                    interpolation = 2,
+                    cosines = [ 1e-11, 1., 20. ],
```

Comment:
This is one of those cases where the same block is used because they are exactly the same in structure but slightly different in interface.

This is the documentation that goes with it:
```
 *  The TabulatedEnergyDistribution class contains the probability
 *  density function (PDF) and cumulative density function (CDF) as a function
 *  of the outgoing energy for the given incident energy or cosine value. It is
 *  used in the OutgoingEnergyDistributionData (ACE LAW 4 for a given incident
 *  energy) and in the OutgoingAngleEnergyDistributionData (ACE LAW 67 for a
 *  given cosine) in the DLW block.
```

I'm not sure if a single keyword difference merits an additional class?